Source/WebCore/platform/graphics/IntRect.h

```diff
@@ -249,6 +249,18 @@ inline IntRect operator-(const IntRect& r, const IntPoint& offset)
     return t -= offset;
 }
 
+inline IntRect& operator+=(IntRect& r, const IntPoint& offset)
```

Comment:
Not sure I agree with this. It's not really clear what it means to add a point and a rect.

Reply:
Is it clear what it means to subtract a point and a rect?? (if the subtraction ones weren't there I would never have done this :P)